app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -12,13 +12,15 @@
 import android.support.v4.content.ContextCompat;
 import android.support.v4.graphics.drawable.DrawableCompat;
 import android.support.v4.view.ViewCompat;
+import android.text.TextUtils;
```

Comment:
Not the android TextUtils, use the app one

Reply:
@iamareebjamal there is only android TextUtils available and no TextUtils function in org.fossasia.openevent.app.common.utils.core. there is only string utils available and StringUtils has no function isEmpty()

Reply:
https://github.com/fossasia/open-event-orga-app/blob/development/app/src/main/java/org/fossasia/openevent/app/common/utils/core/Utils.java#L28